Formula/l/llama-cpp.rb

```diff
@@ -0,0 +1,34 @@
+class LlamaCpp < Formula
+  desc "LLM inference in C/C++"
+  homepage "https://github.com/ggerganov/llama.cpp"
+  url "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b2780.tar.gz"
+  sha256 "8b1831b605ee4c564ff84e648dabdc11fe3d77dcb6b1d7e4d97a14ea991755f2"
+  license "MIT"
+  head "https://github.com/ggerganov/llama.cpp.git", branch: "master"
+
+  livecheck do
+    throttle 10
+  end
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Comment:
It'd be worth it to compile with `DLLAMA_CURL=ON` flag as it allows for users to download a checkpoint via CURL.